ext/ddtrace.stub.php

```diff
@@ -24,6 +24,8 @@
     const DBM_PROPAGATION_FULL = UNKNOWN;
 
     class SpanEvent implements \JsonSerializable {
+        public function __construct(string $name, int $timestamp = null, array $attributes = []) {}
```

Comment:
Let's put timestamp as third arg.

Reply:
Since the OpenTelemetry SDK has the parameter orders of $name, $timestamp, $attributes I wonder if it would make sense to keep this code as-is to match that order: https://github.com/open-telemetry/opentelemetry-php/blob/17ad56a3f3a64481b6764bb46b5584d7bbc5b138/src/SDK/Trace/Event.php#L12

Reply:
But it has no defaults though. Their actual user-facing API has the expected order:
https://github.com/open-telemetry/opentelemetry-php/blob/main/src/SDK/Trace/Span.php#L172